Assets/Scripts/Advertisements/AdvertisementInitializer.cs

```diff
@@ -0,0 +1,47 @@
+using UnityEngine.Advertisements;
+using UnityEngine;
+
+namespace UntitledCube.Advertisements
+{
+    public class AdvertisementInitializer : MonoBehaviour, IUnityAdsInitializationListener
+    {
+        [SerializeField] bool _testMode = true;
```

Comment:
Can this go?

Reply:
No this is the develop mode for the ads, when this is turned off it will show real ads otherwise it shows testing ads